RocketApp/app/src/main/java/com/example/rocketapp/MeasurementTrial.java

```diff
@@ -0,0 +1,20 @@
+package com.example.rocketapp;
+
+import com.google.firebase.firestore.Exclude;
+
+public class MeasurementTrial extends Trial {
+
+    private float measurement;
+
+    public MeasurementTrial() {
+        measurement = 0;
+    }
+
+    public void addMeasurement(){
+        measurement = measurement + 1;
```

Comment:
I think you can just have the measurement passed into the constructor here, since one measurement trial should just hold one float value

Reply:
You're right. For now I have passed it in the function, but I'll add a parameterized constructor as well.